src/components/interfaces/HMI_API.xml

```diff
@@ -503,6 +503,45 @@
     <element name="turnText"/>
 </enum>
 
+<enum name="TextFieldType">
+  <element name="mediaTitle">
+    <description>The data in this field contains the title of the currently playing audio track.</description>
+  </element>
+  <element name="mediaArtist">
+    <description>The data in this field contains the artist or creator of the currently playing audio track.</description>
+  </element>
+  <element name="mediaAlbum">
+    <description>The data in this field contains the album title of the currently playing audio track.</description>
+  </element>
+  <element name="mediaYear">
+    <description>The data in this field contains the creation year of the currently playing audio track.</description>
+  </element>
+  <element name="mediaGenre">
+    <description>The data in this field contains the genre of the currently playing audio track.</description>
+  </element>
+  <element name="mediaStation">
+    <description>The data in this field contains the name of the current source for the media.</description>
+  </element>
+  <element name="rating">
+    <description>The data in this field is a rating.</description>
+  </element>
+  <element name="currentTemperature">
+    <description>The data in this field is the current temperature.</description>
+  </element>
+  <element name="maximumTemperature">
+    <description>The data in this field is the maximum temperature for the day.</description>
+  </element>
+  <element name="minimumTemperature">
+    <description>The data in this field is the minimum temperature for the day.</description>
+  </element>
+  <element name="weatherTerm">
+    <description>The data in this field describes the current weather (ex. cloudy, clear, etc.).</description>
+  </element>
+  <element name="humidity">
+    <description>The data in this field describes the current humidity value.</description>
+  </element>
```

Comment:
@jacobkeeler I meant more from a technical perspective. Not sure how to get to this pull request, i.e. from what to fork/branch off and what to do technically to create this proposal.

Anyways - in the meantime, I looked a bit closer into the "none" issue. Your argument that "proxy implementations would expect the field to exist" seems not correct. The "none" parameter was removed from the HMI_API - it never existed on the MOBILE_API (proxy implementations should not be affected by this).

Do you think, this would still require an additional proposal to handle?

Thanks for your support!